typescript/packages/toolshed/routes/ai/spell/behavior/effects.ts

```diff
@@ -31,7 +31,8 @@ export async function getAllMemories(
 
   const memoryMap: { [key: string]: any } = {};
   memories.forEach((memory) => {
-    const value = memory.is?.value.argument;
+    // FIXME(ja): using jumble can result in memory.is == {}
+    const value = memory.is?.value?.argument;
```

Comment:
@bfollington not sure what's happening here and if this will just cause silent failures ...

without this when using jumble (even just opening a charm) I end up seeing memory.is of `{}`

could be related to trying to open the smol iframe recipes that have an imported html snippet